ui/src/main/webapp/picsureui/styles.css

```diff
@@ -0,0 +1,841 @@
+#main-content {
```

Comment:
We should not need to override this huge file.  Please add any styles needed for the new pages in the styles override file, https://github.com/hms-dbmi/pic-sure-gic-institution/blob/main/ui/src/main/webapp/picsureui/overrides/styles.css